letsencrypt.lua

```diff
@@ -82,7 +82,7 @@ local http_request = function(uri, post_body, options)
     local res, err = httpc:request_uri(uri, defoptions)
 
     if not res then
-        return nil, 500, res.headers, "failed to request: " .. tostring(err)
```

Comment:
Yes, that. So instead of returning an error object we threw the lua euivalent of NullPointerException :)